iroha/src/torii.rs

```diff
@@ -21,25 +21,25 @@ use tokio::{sync::RwLock, task};
 use crate::{
     block_sync::message::VersionedMessage as BlockSyncVersionedMessage,
     config::Configuration,
-    event::{Consumer, EventsReceiver, EventsSender},
+    event::{Consumer, EventsReceiver},
     maintenance::{Health, System},
     prelude::*,
     queue::{GetPendingTransactions, QueueTrait},
-    smartcontracts::isi::query::VerifiedQueryRequest,
+    smartcontracts::{isi::query::VerifiedQueryRequest, permissions::IsQueryAllowedBoxed},
     sumeragi::{
         message::VersionedMessage as SumeragiVersionedMessage, GetLeader, IsLeader, SumeragiTrait,
     },
     wsv::WorldTrait,
 };
 
 /// Main network handler and the only entrypoint of the Iroha.
-#[derive(Debug)]
+#[allow(missing_debug_implementations)]
```

Comment:
May be we can disable this lint, if it doesnt work for generics?

Reply:
Ok